web/tests/narrow_state.test.js

```diff
@@ -334,4 +338,5 @@ test("pm_ids_string", () => {
 
     set_filter([["dm", "bob@foo.com,alice@foo.com"]]);
     assert.equal(narrow_state.pm_ids_string(), "444,555");
+    assert.deepStrictEqual(narrow_state.pm_ids_set(), new Set([444, 555]));
```

Comment:
@evykassirer I added some quick tests for this part while I was working on this.

Reply:
thanks!